### PR TITLE
Remove mention of 200 response to 202-digest request

### PR DESCRIPTION
**Pull request by zimeon**
Attempt at minimal change to address the agreement (?) part of #33.

### CODE REVIEW

**Comment by awoods** [comment]
:+1:

File: index.html [hunk 1]
```diff
@@ -685,21 +685,15 @@ <h2 class='informative'>Implementation Patterns</h2>
           <p>
             If the compared digests do not match, an implementing server MUST reject the request with
             412 Precondition Failed. If the compared digests match, an implementing server MUST complete
-            the request with 202 Accepted. A 200 OK status indicates that the server did not understand
-            the request, and ignored the expectation.
+            the request with 202 Accepted. A client that receives a non-202 response to a
+            <code>202-digest</code> expectation should assume it is responsible for verifying fixity by
```
Comment (acoburn) [inline index.html]: s/should/SHOULD/ ?

Comment (zimeon) [inline index.html]: Done

**Review by acoburn — APPROVED** [review]

**Comment by acoburn** [comment]
👍

File: index.html (hunk 1 above)
Comment (acoburn) [inline index.html]: Thanks @zimeon

**Pull request merged**